app.js

```diff
@@ -1,6 +1,6 @@
 const { getWaifu } = require('./utils/puppeteerScrapper')
 const weebhook = require('./utils/webhook')
 
-getWaifu().then(waifu => {
-  weebhook.sendDiscordMessage(waifu)
-})
+getWaifu()
+  .then(waifu => weebhook.sendDiscordMessage(waifu))
+  .then(() => process.exit())
```

Comment:
Why this is necessary?... the process will end here anyways if the promise is resolved.

Reply:
Process wasn't ending by itself on my environment, will remove the line if its not an issue then.

Reply:
Hmm... if that's the case then i think there might be a puppeteer thread that never ends. We probably aren't disposing correctly the instance. I'll take a look at the docs to see if i can find something about it.